api/proxy/Makefile

```diff
@@ -27,20 +27,12 @@ disperse-test-blob:
 test-unit:
 	gotestsum --format pkgname-and-test-fails -- -short -parallel 4 ./...
 
-# E2E tests using local memstore, leveraging op-e2e framework. Also tests the standard client against the proxy.
+# E2E tests using local memstore. Also tests the standard client against the proxy.
 test-e2e-local:
 	BACKEND=memstore gotestsum --format testname -- -v -timeout 10m ./test/e2e -parallel 8
 
-# E2E tests using holesky testnet backend, leveraging op-e2e framework. Also tests the standard client against the proxy.
-# If holesky tests are failing, consider checking https://dora.holesky.ethpandaops.io/epochs for block production status.
-test-e2e-testnet:
-	BACKEND=testnet gotestsum --format testname -- -v -timeout 20m ./test/e2e -parallel 32
-
-## Equivalent to `test-e2e-testnet`, but against preprod instead of testnet
-test-e2e-preprod:
-	BACKEND=preprod gotestsum --format testname -- -v -timeout 20m ./test/e2e -parallel 32
```

Comment:
shouldnt we keep these and instead migrate them to use the new hoodi networks instead?

Reply:
[05ac760](https://github.com/Layr-Labs/eigenda/pull/2245/commits/05ac760c75622aa16a8536bac311af7bffce14c8)